src/FileSystem-Disk/FilePluginPrims.class.st

```diff
@@ -33,10 +33,10 @@ FilePluginPrims >> connectToFile: filePointer writable: writableFlag [
 
 	<primitive: 'primitiveConnectToFile' module: 'FilePlugin' error: error>
 	error = #'bad argument' ifTrue: [ 
-		filePointer isBytes ifFalse: 
-			[ ^self error: 'openFileFile:writable: filePointer must be bytes' ].
+		(filePointer isKindOf: ByteArray) ifFalse: 
```

Comment:
Not sure this change was intended? Or does filePointer does not understand #isBytes?

Reply:
It does understand isBytes, but Strings also answer true to isBytes, so this is slightly tighter parameter checking.  There's no chance for polymorphism in the primitive, it really is expecting a buffer of bytes containing a C pointer.